packages/edge-login-ui-rn/src/native/components/exportAPIComponents/LoginScreen.js

```diff
@@ -44,22 +45,29 @@ class LoginScreen extends Component<Props> {
       typeof window === 'object' && window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
         ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({ name: 'core-ui' })
         : compose
+
+    // Hack to make Flow happy, since it expects these objects to exist:
+    const missing: any = void 0
+    // Another hack, since `recoveryKey` is `string`
+    // but `recoveryLogin` is `boolean`:
+    const recoveryKey: any = this.props.recoveryLogin
```

Comment:
This is worrysome. The types simply don't match up. Is it a `boolean` or a `string`? I didn't see where this was used, so I couldn't figure it out.